safe/impact_function/multi_exposure_wrapper.py

```diff
@@ -471,6 +503,9 @@ def _generate_provenance(self):
             provenance_data_store_uri,
             self.datastore.uri_path)
 
+        # Map title
+        set_provenance(self._provenance, provenance_map_title, self.name)
```

Comment:
Side question, is it not already `provenance_impact_function_name`?

Reply:
yes, but I need map title provenance, and I can't think something better than the name to be the map title

Reply:
Ok, no problem, let's do that until they want a better naming